### PR TITLE
More improvements to bootstrap

### PR DESCRIPTION
**Pull request by bfcoder**
Don't symlink anymore, just copy the example file.

### CODE REVIEW

**Review by dspencer001 — APPROVED** [review]
LGTM

**Review by conradbeach — APPROVED** [review]

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -32,7 +32,7 @@ rspec:
   services:
     - postgres:9.5.0
   before_script:
-    - cp config/secrets.example.yml config/secrets.yml
```
Comment (dspencer001) [inline .gitlab-ci.yml:35]: The file is named secrets.example.yml - are you renaming it?

**Review by srphillips — APPROVED** [review]
I approve

File: .gitlab-ci.yml (hunk 1 above)
Comment (dspencer001) [reply .gitlab-ci.yml:35]: oh, yes, you did. Nevermind.

Comment (bfcoder) [reply .gitlab-ci.yml:35]: yes

**Comment by oxtralite** [comment]
Does this check to see if it already exists before copying it? Please avoid
overwriting existing ones

**Comment by bfcoder** [comment]
> Does this check to see if it already exists before copying it? Please avoid overwriting existing ones

@oxtralite Yes, it does.

**Review by jbasdf — APPROVED** [review]
Much better.

**Pull request merged**